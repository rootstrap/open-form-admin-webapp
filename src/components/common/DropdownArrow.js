import React from 'react';

const DropdownArrow = props => (
  <svg
    width="15"
    height="8"
    viewBox="0 0 15 8"
    fill="none"
    xmlns="http://www.w3.org/2000/svg"
    {...props}
  >
    <path
      d="M1.48242 1L7.48242 7L13.4824 1"
      stroke="#4F4F4F"
      strokeWidth="2"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
  </svg>
);

export default DropdownArrow;
