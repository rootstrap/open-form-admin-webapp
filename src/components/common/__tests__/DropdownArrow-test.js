import React from 'react';
import renderer from 'react-test-renderer';

import DropdownArrow from '../DropdownArrow';

it('renders correctly', () => {
  const tree = renderer.create(<DropdownArrow />).toJSON();
  expect(tree).toMatchSnapshot();
});
