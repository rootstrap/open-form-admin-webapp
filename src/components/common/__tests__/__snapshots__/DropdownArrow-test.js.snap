// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  fill="none"
  height="8"
  viewBox="0 0 15 8"
  width="15"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M1.48242 1L7.48242 7L13.4824 1"
    stroke="#4F4F4F"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
  />
</svg>
`;
